src/test/unit/sanitize.test.js

```diff
@@ -29,17 +29,22 @@ import {
 
 describe('sanitizePII', function () {
   function setup(
-    piiConfig,
+    piiConfig: $Shape<RemoveProfileInformation>,
```

Comment:
Probably it could be harder to do on a unit test, but I would like to make sure that we don't accidentally remove the other tab information from the profile when user is not on active tab view and when they are on release (since the default is to sanitize by default on release). Could you also add a test for that? (I couldn't find one for that, let me know if there is)